cdc/generation.hh

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2019 ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <vector>
+
+#include "bytes.hh"
+#include "database_fwd.hh"
+
+namespace ser {
+
+template <typename T>
+class serializer;
+
+}
+
+namespace dht {
+
+class token;
+
+} // namespace dht
+
+namespace cdc {
+
+class stream_id final {
```

Comment:
I sure wish there was a `docs/cdc.md` that would explain what these streams are. No need to add to this series, but would be nice to add as a follow-up.

Reply:
I added it